debugd/internal/debugd/metadata/cloudprovider/cloudprovider.go

```diff
@@ -91,18 +91,7 @@ func (f *Fetcher) DiscoverLoadbalancerIP(ctx context.Context) (string, error) {
 }
 
 // FetchSSHKeys will query the metadata of the current instance and deploys any SSH keys found.
+// TODO: remove completely or re-implement to be independent of the internal cloud package.
 func (f *Fetcher) FetchSSHKeys(ctx context.Context) ([]ssh.UserKey, error) {
-	self, err := f.metaAPI.Self(ctx)
-	if err != nil {
-		return nil, fmt.Errorf("retrieving ssh keys from cloud provider metadata: %w", err)
-	}
-
-	keys := []ssh.UserKey{}
-	for username, userKeys := range self.SSHKeys {
-		for _, keyValue := range userKeys {
-			keys = append(keys, ssh.UserKey{Username: username, PublicKey: keyValue})
-		}
-	}
-
-	return keys, nil
+	return nil, nil
```

Comment:
In case this shouldn't be called but is still needed for some interface, please panic instead of returning nil. Otherwise I'd suggest to delete.

Reply:
Will be removed by https://github.com/edgelesssys/constellation/pull/470